src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/adjudications/AdjudicationsService.kt

```diff
@@ -418,14 +411,6 @@ class AdjudicationsService(
     }
   }
 
-  // TODO future change Remove logging and use value only from mapping service once satisfied location picked up correctly
```

Comment:
Monitoring of this over the last month has shown only connectivity issues (i.e. no missing location ids) so can now be removed